<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>895</width>
    <height>510</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>XORFileProcessor</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../../Downloads/electrical.png</normaloff>../../../Downloads/electrical.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>841</width>
      <height>421</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0,0,0,0,0,0,0,0">
     <property name="sizeConstraint">
      <enum>QLayout::SizeConstraint::SetNoConstraint</enum>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>7</number>
       </property>
       <property name="sizeConstraint">
        <enum>QLayout::SizeConstraint::SetNoConstraint</enum>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="inputPathLabel">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Папка для обработки:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="selectInputButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="baseSize">
          <size>
           <width>2</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>Обзор</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="inputDirLineEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="maskLabel">
         <property name="text">
          <string>Маска для файлов:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="maskLineEdit">
         <property name="text">
          <string>*</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <property name="spacing">
        <number>9</number>
       </property>
       <property name="sizeConstraint">
        <enum>QLayout::SizeConstraint::SetDefaultConstraint</enum>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Введите 8-байтный ключ (8 символов):</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="keyLineEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QCheckBox" name="timerCheckBox">
         <property name="text">
          <string>Работа по таймеру</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="timeLabel">
         <property name="text">
          <string>Периодичность:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTimeEdit" name="dateEdit">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="maximumTime">
          <time>
           <hour>23</hour>
           <minute>59</minute>
           <second>59</second>
          </time>
         </property>
         <property name="displayFormat">
          <string>mm дней</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTimeEdit" name="timeEdit">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="displayFormat">
          <string>HH часов mm минут ss секунд</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="stopTimerButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Остановить таймер</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="topMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Папка для сохранения:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="browseOutputButton">
         <property name="text">
          <string>Обзор</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="outputDirLineEdit">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QCheckBox" name="overWriteCheckBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Перезаписывать существующие файлы</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="deleteInputCheckBox">
       <property name="text">
        <string>Удалять исходные файлы</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="processButton">
       <property name="text">
        <string>Обработать</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QProgressBar" name="progressBar">
       <property name="value">
        <number>24</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>895</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
